src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tab.js

```diff
@@ -1,9 +1,11 @@
 // @flow
 import React from 'react';
+import type {Element} from 'react';
 import classNames from 'classnames';
 import tabStyles from './tab.scss';
 
 type Props = {
+    badges: ?(Element<*>[] | Element<*>),
```

Comment:
think allowing `Element<*>` instead of just allowing an array makes it harder to not render the container below 🙂